accre-internal/Singularity.accre.internal.base

```diff
@@ -7,15 +7,34 @@ Include: yum
   Maintainer Eric Appelt
 
 %help
-  Base container for internal ACCRE use with LMod support
-  To access the software stack make sure to bind mount /cvmfs and /accre
+  Base container for internal ACCRE use with Lmod support
+  To access the software stack make sure to bind mount /accre/arch and /accre/common
 
-%files
-  lmod.sh /etc/profile.d/lmod.sh
-  lmod-7.8.8-1.x86_64.rpm /lmod-7.8.8-1.x86_64.rpm
+%environment
+  export MODULEPATH_ROOT=/accre/arch/easybuild/modules/all
+  export MODULEPATH=$MODULEPATH_ROOT/Core:$MODULEPATH_ROOT/BinDist
+  export LMOD_SHORT_TIME=86400
+  export LMOD_ADMIN_FILE=/accre/common/lmod/etc/admin.lmod
+  export LMOD_RC=/accre/common/lmod/etc/lmodrc.lua
+  source /usr/local/lmod/lmod/init/bash
 
 %post
   yum -y install wget bzip2 gcc tcl tk tar procps git
+  
+  # Install Lmod
+  export LMOD_VERSION=8.1.7
   yum -y install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
-  yum -y install lua lua-devel lua-posix lua-filesystem
-  yum -y install /lmod-7.8.8-1.x86_64.rpm
+  yum -y install lua lua-filesystem lua-posix lua-term lua-devel
+  yum -y install make
+  wget https://github.com/TACC/Lmod/archive/${LMOD_VERSION}.tar.gz
```

Comment:
Glad to see we don't have to pull from sourceforge!

Reply:
I hate SF